src/configLoaders/ReactConfigLoader.ts

```diff
@@ -0,0 +1,35 @@
+import ConfigLoader from './ConfigLoader';
+import createReactJestConfig from '../utils/createReactJestConfig';
+import * as path from 'path';
+import JestConfiguration from './JestConfiguration';
+
+export default class ReactConfigLoader implements ConfigLoader {
```

Comment:
I'm not so happy with this file / class name. It suggests you need this when you use React, which is not true. You need it when you use [Create React App (CRA)](https://github.com/facebookincubator/create-react-app). Maybe we should name it `ReactScriptsLoader` (because CRA uses `react-scripts` under the hood).

Reply:
Well it isn't a loader for React scripts either. What about `ReactJestConfigLoader`, maybe `ConfigLoader` has to be renamed to `JestConfigLoader` and `DefaultConfigLoader` to `DefaultJestConfigLoader`. That way we clarify that it is loading a Jest configuration.